### ISSUE TITLE
Info section is invisible in the document outline on story pages

### ISSUE THREAD

**Issue by rosejbon** [neontribe/contemplating-action#238]
## Description
Some sections, while made distinct with visual styling and HTML5 `<section>` tags, are left without headings. This leaves them invisible and unlabelled in the document outline. For each section, it should be considered as to whether they would be deserving of a place in the outline. e.g.
      * The information given after a story:
        ![image](https://user-images.githubusercontent.com/2150762/63610714-fd4c8d00-c5d1-11e9-91b9-f6bf62d05759.png)

Need to think about whether people may want to skip straight to the info instead of reading the stories.
